Makefile

```diff
@@ -1,4 +1,4 @@
-BLACK_CHECK = black -l 90 --check --diff .
+BLACK_CHECK = black -l 90 --diff .
```

Comment:
Did you mean to commit this? This doesn't seem needed in this PR.

Reply:
Actually, the presence of the `--check` flag make the Black formatter to exit if there is a formatting issue. I removed it since so that instead of exiting, it format the files with issue to conform to its formatting rules. I will re-add it back. Perhaps, it can be addressed in a different PR or issue?

Reply:
We have a `make` command to format the files, you can run this before you push your changes.